backend/src/langchain.ts

```diff
@@ -39,7 +39,7 @@ function getChatModel() {
 
 function initQAModel(level: LEVEL_NAMES, Prompt: string) {
 	const openAIApiKey = getOpenAIKey();
-	const documentVectors = vectorisedDocuments.get()[level].docVector;
+	const documentVectors = getDocumentVectors()[level].docVector;
```

Comment:
This is fine, though as you are only ever using getDocumentVectors like this, you _could_ modify the get function itself to take the level param and return the docVector. Small beans though, and might mean reworking some tests, so up to you.

Reply:
Had a go and wasn't straightforward so I'm gonna go ahead and ignore it